### DISCUSSION TITLE
continuous restore of incremental backups

### DISCUSSION THREAD

**Discussion by drenton1** [abbbi/virtnbdbackup#100] [Q&A / Support] [answered]
Thanks for this great program.  I'm successfully backing up and restoring virtual machines.  Some of our VMs are quite large so I'm trying to improve the restore time.  This is what I'm trying to do.

Sunday - create full backup, after the backup is done, run the virtnbdrestore command
Monday - create an incremental backup and use virtnbdrestore to merge this incremental backup with the Sunday restored qcow2

I tried using the --until and the --sequence but they are meant to be used with the full backup.  Essentially I'm trying to restore the full backup just once per week and apply the incrementals daily.

Thanks

**Comment by abbbi** [comment, marked as answer]
its currently not possible to apply incremental backups to already created restores. I get what you are trying to archive, but without implementing this in virtnbdrestore, this isn't possible at the moment.
